discord_bots/cogs/admin.py

```diff
@@ -0,0 +1,692 @@
+                            colour=Colour.blue(),
+                        ),
+                        ephemeral=True,
+                    )
+
+    @group.command(name="removecommand", description="Remove a custom command")
+    @app_commands.check(is_admin_app_command)
+    @app_commands.check(is_command_channel)
+    @app_commands.describe(name="Name of existing custom command")
+    async def removecommand(self, interaction: Interaction, name: str):
+        session: SQLAlchemySession
+        with Session() as session:
+            exists = (
+                session.query(CustomCommand).filter(CustomCommand.name == name).first()
+            )
+            if not exists:
+                await interaction.response.send_message(
+                    embed=Embed(
+                        description="Could not find command with that name",
+                        colour=Colour.red(),
+                    ),
+                    ephemeral=True,
+                )
+                return
+
+            session.delete(exists)
+            session.commit()
+
+            await interaction.response.send_message(
+                embed=Embed(
+                    description=f"Command `{name}` removed",
+                    colour=Colour.green(),
+                )
+            )
+
+    @group.command(name="removedbbackup", description="Remove a database backup")
+    @app_commands.check(is_admin_app_command)
+    @app_commands.check(is_command_channel)
+    @app_commands.describe(db_filename="Name of backup file")
+    async def removedbbackup(self, interaction: Interaction, db_filename: str):
+        if not db_filename.startswith(config.DB_NAME) or not db_filename.endswith(
+            ".db"
+        ):
+            await interaction.response.send_message(
+                embed=Embed(
+                    description=f"Filename must be of the format {config.DB_NAME}_{{date}}.db",
+                    colour=Colour.red(),
+                ),
+                ephemeral=True,
+            )
+            return
+
+        try:
+            os.remove(db_filename)
+        except Exception as e:
+            _log.exception(f"Caught Exception in removedbbackup: {e}")
+            await interaction.response.send_message(
+                embed=Embed(
+                    description=f"Failed to remove DB Backup {db_filename}",
+                    colour=Colour.red(),
+                ),
+                ephemeral=True,
+            )
+        else:
+            await interaction.response.send_message(
+                embed=Embed(
+                    description=f"DB backup {db_filename} removed",
+                    colour=Colour.green(),
+                )
+            )
+
+    @group.command(name="restart", description="Restart the bot")
+    @app_commands.check(is_admin_app_command)
+    @app_commands.check(is_command_channel)
+    async def restart(self, interaction: Interaction):
```

Comment:
This might be dangerous to keep around with the dockerized setup, since any admin can use it as well. I'm not sure how it behaves with docker, though

Reply:
The early return on postgres db's will be sufficient for now to block any bad interactions. In future this & `createdbbackup()` should be updated to support the dockerized db.
`TODO` added